src/Components/NavBar/NavBar.js

```diff
@@ -20,11 +20,11 @@ const NavBar = (props) => {
 					<li>
 						<NavLink to="/trips">Trips</NavLink>
 					</li>
-					{context.token && (
-						<li>
-							<NavLink to="/bookings">Bookings</NavLink>
-						</li>
-					)}
+
```

Comment:
It's only the trips route that doesn't require authentication. And that's because I want a user to be able to view the available trips even if he is not yet registered. But they cannot book a trip without being registered